src/data/1-312-data.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS officers (
```

Comment:
:sparkles: :tada: Perfect! Do you mind moving this into a `migrations` subfolder, and adding the date to the filename? This helps for future migrations.